src/doc/src/reference/unstable.md

```diff
@@ -842,6 +842,13 @@ crates, which can save significant time and bandwidth.
 
 The format of the sparse index is identical to a checkout of a git-based index.
 
+The `registries.crates-io.protocol` config option can be used to set the default protocol
+for crates.io. This option requires `-Z sparse-registry` to be enabled.
+
+* `sparse` — Use sparse index.
+* `git` — Use git index.
+* If the option is unset, it will be sparse index if `-Z sparse-registry` is enabled, otherwise it will be git index.
```

Comment:
Does this make sense? Because for now, if the option is unset, it always is the sparse index.